eng/Versions.props

```diff
@@ -91,15 +91,42 @@
          project references for command line builds. They should match the values in the diagnostics repo. -->
     <MicrosoftBclAsyncInterfacesVersion>6.0.0</MicrosoftBclAsyncInterfacesVersion>
     <MicrosoftDiagnosticsTracingTraceEventVersion>3.0.7</MicrosoftDiagnosticsTracingTraceEventVersion>
-    <MicrosoftExtensionsLoggingVersion>7.0.0</MicrosoftExtensionsLoggingVersion>
+  </PropertyGroup>
+  <PropertyGroup>
+    <!--
+      This property sets a baseline version for packages produced by the dotnet/runtime repository such that
+      the value is the same as that in the SDK OR OLDER. This property can be used for packages that ship as
+      part of the corresponding framework; this allows those packages to be referenced but not copied to the
+      output directory since the ref pack from the SDK should be the same version or newer (this causes a package
+      version conflict and will result in the framework reference winning; since it is part of the framework,
+      the assembly is not copied to the output directory). However, this cause the NuGet package dependency
+      for any NuGet packages that are created have this as the minimum version.
```

Comment:
One confusing element for this explanation is why the reference is required at all? If the package is already part of the runtime, is PackageReference even necessary if we do not need a higher version?

Reply:
You're right, but this is the behavior that I was seeing for some reason.

I think I'm going to merge this in as-is in order to unblock the dotnet/installer updates and then do more investigation as a follow up.